{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2016-6225",
      "ASSIGNER": "cve@mitre.org"
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "CWE-326"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "https://www.percona.com/blog/2017/01/12/cve-2016-6225-percona-xtrabackup-encryption-iv-not-set-properly/",
          "name": "https://www.percona.com/blog/2017/01/12/cve-2016-6225-percona-xtrabackup-encryption-iv-not-set-properly/",
          "refsource": "CONFIRM",
          "tags": [
            "Vendor Advisory"
          ]
        },
        {
          "url": "https://github.com/percona/percona-xtrabackup/pull/267",
          "name": "https://github.com/percona/percona-xtrabackup/pull/267",
          "refsource": "CONFIRM",
          "tags": [
            "Issue Tracking",
            "Patch",
            "Third Party Advisory"
          ]
        },
        {
          "url": "https://github.com/percona/percona-xtrabackup/pull/266",
          "name": "https://github.com/percona/percona-xtrabackup/pull/266",
          "refsource": "CONFIRM",
          "tags": [
            "Issue Tracking",
            "Patch",
            "Third Party Advisory"
          ]
        },
        {
          "url": "https://bugs.launchpad.net/percona-xtrabackup/+bug/1643949",
          "name": "https://bugs.launchpad.net/percona-xtrabackup/+bug/1643949",
          "refsource": "CONFIRM",
          "tags": [
            "Issue Tracking",
            "Patch",
            "Third Party Advisory"
          ]
        },
        {
          "url": "http://lists.opensuse.org/opensuse-updates/2017-01/msg00126.html",
          "name": "openSUSE-SU-2017:0251",
          "refsource": "SUSE",
          "tags": [
            "Third Party Advisory"
          ]
        },
        {
          "url": "http://lists.opensuse.org/opensuse-updates/2017-01/msg00125.html",
          "name": "openSUSE-SU-2017:0250",
          "refsource": "SUSE",
          "tags": [
            "Third Party Advisory"
          ]
        },
        {
          "url": "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/ZBVCP6KLFVGG6HSGLHLTMZRD6C4IJSZP/",
          "name": "FEDORA-2017-5a823376be",
          "refsource": "",
          "tags": []
        },
        {
          "url": "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/BAHI6ETS22FJCMLW7A6SICFKQXF5G2VI/",
          "name": "FEDORA-2017-6382ea8d57",
          "refsource": "",
          "tags": []
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "xbcrypt in Percona XtraBackup before 2.3.6 and 2.4.x before 2.4.5 does not properly set the initialization vector (IV) for encryption, which makes it easier for context-dependent attackers to obtain sensitive information from encrypted backup files via a Chosen-Plaintext attack. NOTE: this vulnerability exists because of an incomplete fix for CVE-2013-6394."
        }
      ]
    }
  },
  "configurations": {
    "nodes": [
      {
        "operator": "OR",
        "cpe_match": [
          {
            "vulnerable": true,
            "cpe23Uri": "cpe:2.3:a:percona:xtrabackup:*:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe23Uri": "cpe:2.3:a:percona:xtrabackup:2.4.1:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe23Uri": "cpe:2.3:a:percona:xtrabackup:2.4.0:rc1:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe23Uri": "cpe:2.3:a:percona:xtrabackup:2.4.3:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe23Uri": "cpe:2.3:a:percona:xtrabackup:2.4.2:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe23Uri": "cpe:2.3:a:percona:xtrabackup:2.4.4:*:*:*:*:*:*:*"
          }
        ]
      },
      {
        "operator": "OR",
        "cpe_match": [
          {
            "vulnerable": true,
            "cpe23Uri": "cpe:2.3:o:opensuse:leap:42.2:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe23Uri": "cpe:2.3:o:opensuse:leap:42.1:*:*:*:*:*:*:*"
          }
        ]
      },
      {
        "operator": "OR",
        "cpe_match": [
          {
            "vulnerable": true,
            "cpe23Uri": "cpe:2.3:o:fedoraproject:fedora:25:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe23Uri": "cpe:2.3:o:fedoraproject:fedora:24:*:*:*:*:*:*:*"
          }
        ]
      }
    ]
  },
  "impact": {
    "baseMetricV2": {
      "cvssV2": {
        "vectorString": "AV:N/AC:M/Au:N/C:P/I:N/A:N",
        "accessVector": "NETWORK",
        "accessComplexity": "MEDIUM",
        "authentication": "NONE",
        "confidentialityImpact": "PARTIAL",
        "integrityImpact": "NONE",
        "availabilityImpact": "NONE",
        "baseScore": 4.3
      },
      "severity": "MEDIUM",
      "exploitabilityScore": 8.6,
      "impactScore": 2.9,
      "obtainAllPrivilege": false,
      "obtainUserPrivilege": false,
      "obtainOtherPrivilege": false,
      "userInteractionRequired": false
    }
  },
  "publishedDate": "2017-03-23T16:59Z",
  "lastModifiedDate": "2023-11-07T02:33Z"
}